src/lib/Notifications.svelte

```diff
@@ -217,13 +217,17 @@
     const computedTimeout = options.persist ? 0 : (options.timeout || timeout)
     const id = Math.random().toString(36).replace(/[^a-z]+/g, '')
 
-    sessionStorage.setItem(
-      sessionKey,
-      JSON.stringify([
-        ...JSON.parse(sessionStorage.getItem(sessionKey) || '[]'),
-        { ...detail, id }
-      ])
-    )
+    try {
+      sessionStorage.setItem(
+        sessionKey,
+        JSON.stringify([
+          ...JSON.parse(sessionStorage.getItem(sessionKey) || '[]'),
+          { ...detail, id }
+        ])
+      )
+    } catch (e) {
+      console.log('unable to set sessionStorage', e)
```

Comment:
we don't even need to warn here - I think I said to you yesterday we shouldn't have empty exception blocks, but I guess this is an exception!